packages/web/src/lib/web-mode.ts

```diff
@@ -149,6 +149,61 @@ export class WebMode {
 		return s;
 	}
 
+	/**
+	 * Append signature multisig transaction using algosigner
+	 * @param txns array of multisignature transaction with at least one signature
+	 * @param signers a subset of addresses to sign with the transaction
+	 * return an object contain blob attribute in base64
+	 */
+	async appendSignMultisigTransaction(txns: WalletTransaction[], signers: string[]): Promise<JsonPayload> {
+		const result:JsonPayload = {};
+		for (let i = 0; i < txns.length; ++i) {
+			const txn = txns[i];
+			const partialTxn = algosdk.decodeObj(
+				this.algoSigner.encoding.base64ToMsgpack(txn.txn)) as EncodedSignedTransaction; 
+			if (partialTxn.txn === undefined || partialTxn.msig === undefined) {
+				throw new Error("Input transaction must be multisigature transaction signed with at least 1 signature");
+			} 
+			const noSigTxn = algosdk.Transaction.from_obj_for_encoding(partialTxn.txn);
```

Comment:
what the `no` stands for?

Reply:
`no` stand for no `msig` in the `noSigTxn`(No signature transaction)
`partialTxn` contains `msig` which is the multsig params and `txn` which is the algosdk.Transaction

Reply:
I think you should rename it. noSigTxnBytes is not sense.

Reply:
But `noSigTxn` is the transaction in types algosdk.Transaction
And `noSigTxnBytes` is the transaction in types uint8array
I name it `Bytes` to recognize that it in uint8array

Reply:
So lets do it more explicit. I think its better use a longer name that the person that reads the code in the future knows exactly what each variable corresponds to.